packages/next-app/src/components/Sidebar.tsx

```diff
@@ -0,0 +1,84 @@
+import { MagnifyingGlassIcon } from "@radix-ui/react-icons";
+import Link from "next/link";
+import { useState } from "react";
+
+import { Actions } from "#/app/daoadmin/(components)/Actions";
+import { ActionAttribute, useAdminTools } from "#/contexts/AdminToolsContext";
+import { hardcodedData } from "#/utils/hardcodedData";
+
+import { FilterDropdown, IFilter } from "./FilterDropdown";
+
+export function Sidebar() {
+  const data = hardcodedData;
+  const [querySearch, setQuerySearch] = useState("");
+  const { selectedAction, handleSetAction, selectedFilters } = useAdminTools();
+
+  const handleSearchChange = (e: React.FormEvent<HTMLInputElement>) => {
+    setQuerySearch(e.currentTarget.value);
+  };
+
+  const filteredActionsByTag =
+    selectedFilters.operationResponsible !== ""
+      ? data.actions.filter(
+          (item) =>
+            item.operationResponsible === selectedFilters.operationResponsible
+        )
+      : data.actions;
+
+  const filteredActionsByName = filteredActionsByTag.filter((item) =>
+    item.name.toLowerCase().includes(querySearch.toLowerCase())
+  );
+
+  const filters: IFilter[] = [
+    {
+      name: "Operation Responsible",
+      options: ["Maxis", "Labs"],
+    },
+  ];
```

Comment:
`#must` refactor this out of the Sidebar component. Also, just as a `meta` comment but I think this is more a list of items (ActionListMenu, if you will) than can be called a Sidebar which is usually alongside the main display area and not conditional on the number of items that are in it. 

Also a very strongly `#would` imo is refactoring this to avoid code duplication between the metadata and daoadmin. Even more considering this is placed in the root `components` folder but it has a bunch of daoadmin logic intertwined.